infoschema/metric_table_def.go

```diff
@@ -0,0 +1,1838 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infoschema
+
+// MetricTableMap records the metric table definition, export for test.
+// TODO: read from system table.
+var MetricTableMap = map[string]MetricTableDef{
+	"query_duration": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tidb_server_handle_query_duration_seconds_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,sql_type,instance))`,
+		Labels:   []string{"instance", "sql_type"},
+		Quantile: 0.90,
+		Comment:  "The quantile of TiDB query durations(second)",
+	},
+	"qps": {
+		PromQL:  `sum(rate(tidb_server_query_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (result,type,instance)`,
+		Labels:  []string{"instance", "type", "result"},
+		Comment: "TiDB query processing numbers per second",
+	},
+	"qps_ideal": {
+		PromQL: `sum(tidb_server_connections) * sum(rate(tidb_server_handle_query_duration_seconds_count[$RANGE_DURATION])) / sum(rate(tidb_server_handle_query_duration_seconds_sum[$RANGE_DURATION]))`,
+	},
+	"ops_statement": {
```

Comment:
`ops_statement`?

Reply:
How about add a `tidb` prefix?
I used to doesn't add `tidb` for TiDB metrics, but PD and TiKV has `pd`/`tikv` prefix.